include/numerics/tensor_value.h

```diff
@@ -275,6 +287,35 @@ TensorValue<T>::TensorValue (const TypeTensor<Real> & p_re,
 }
 #endif
 
+template <typename T>
+TensorValue<T>
+TensorValue<T>::rotation_matrix(const Real phi, const Real theta, const Real psi)
+{
+#if LIBMESH_DIM == 3
+  const Real p = -phi / 180. * libMesh::pi;
```

Comment:
```suggestion
  const Real p = -phi / 180. * pi;
```
Hmm I set about writing the inverse rotation and now am in this quandary as to why the normal rotation has negative signs applied here

Reply:
@dknez I think you were the one who originally added the code in `MeshModification::rotate` (way back in 2011 it looks like). I've checked and these negative signs are indeed necessary in order to get counter-clockwise/right-hand-rule rotation. I feel like this means there is an error/omission in the [Wolfram documentation](https://mathworld.wolfram.com/EulerAngles.html) or maybe I am missing a detail in the documentation?

Reply:
hmm, good question... I don't remember the details here, sorry!

Reply:
Looks like code was originally added in 476198bb789923932767df4fd5884dc759b2113a by @benkirk and possibly @MartinLuethi was also involved. I concur that the Wolfram article seems to be wrong, and we have been using a corrected version in our code all these years. A better reference might be the [Wikipedia](https://en.wikipedia.org/wiki/Euler_angles#Rotation_matrix) article, specifically the last row of the table in the "Proper Euler angles" column.